pydantic/errors.py

```diff
@@ -337,3 +337,7 @@ class IPv6InterfaceError(PydanticValueError):
 
 class ColorError(PydanticValueError):
     msg_template = 'value is not a valid color: {reason}'
+
+
+class StrictBoolError(PydanticValueError):
+    msg_template = 'value is not a strict bool'
```

Comment:
```suggestion
    msg_template = 'value is not a valid boolean value'
```

Reply:
This doesn't improve clarity. `value is not a ... value`...

Reply:
Okay, "value is not a boolean"